resources/tap.brs

```diff
@@ -95,9 +95,16 @@ sub __tap_printExtras(extra = {}, level = 0)
     m.indent()
     for each item in extra
         if type(extra[item]) = "roAssociativeArray" then
-            print m.getIndent() item ":"
             level = level + 1
-            m.printExtras(extra[item], level)
+            if extra[item]._roca_isMultilineString = true
```

Comment:
🤔 where does this get set?  I can't find it 🔎

Reply:
It gets set here: https://github.com/hulu/roca/pull/77/files#diff-322cfb6820ddfa62cfd39e2c232fd4aa7257650fb2177163216038c01402613bR84

Reply:
it's a little funky... but i'm not sure how else to render multiline in only these scenarios

Reply:
👓 I swear I command-f'd for that 🤦

Reply:
although we could always render _every_ string as multiline? the raw YAML output would just be harder to read.

Reply:
> although we could always render _every_ string as multiline? the raw YAML output would just be harder to read.

That's certainly an option.  I'm fine with either!